src/util/updates.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+import defaultUpdateNotifier from 'update-notifier';
+
+type checkForUpdatesParams = {
+  version: string,
+  updateNotifier?: typeof defaultUpdateNotifier,
+};
+
+export function checkForUpdates({
+  version,
+  updateNotifier = defaultUpdateNotifier,
+}: checkForUpdatesParams
```

Comment:
nit: I think it would be easier to visualize the nesting of these statements if they were indented one more level like:
````javascript
export function checkForUpdates(
  {
    version,
    updateNotifier = defaultUpdateNotifier,
  }: checkForUpdatesParams
) {
  ...
}
````